### PR TITLE
[BYTEMAN-414] speed up compilation to skip tests and doc generation

### PR DESCRIPTION
**Pull request by ochaloup**
https://issues.redhat.com/browse/BYTEMAN-414

### CODE REVIEW

**Comment by ochaloup** [comment]
@adinn an attempt to address the enhancement for compilation to skip tests and doc generation

File: README [hunk 1]
```diff
@@ -66,6 +66,17 @@ or application code.
...
+Compiling Byteman
+-----------------
+
+To compile source code use Maven command `mvn clean install`.
+Byteman will be compiled, the command generates documentation
+and distribution artifacts can be found at `download/target`.
+
+To speed-up compilation you can skip tests and generating
+the documentation with
+`mvn clean install -DskipITs -DskipTests -DnoDoc`.
```
Comment (adinn) [inline README:78]: This is a very nice idea. It might be better to use -DskipDocs instead of -DnoDocs.

File: agent/pom.xml [hunk 2]
```diff
@@ -2376,7 +2376,7 @@
                 <artifactId>maven-verifier-plugin</artifactId>
                 <configuration>
                     <verificationFile>verification.xml</verificationFile>
-                    <failOnError>true</failOnError>
+                    <failOnError>${verificationFailOnError}</failOnError>
                 </configuration>
```
Comment (adinn) [inline agent/pom.xml:2380]: Perhaps a better alternative would be to have an empty verification file (noverification.xml), default the filename to verification.xml and reset it int the skipITs profile.

**Review by adinn — CHANGES_REQUESTED** [review]
This is nice but can you consider the alternatives I suggested? If you think your solution is better I'm happy to go with it.

File: README (hunk 1 above)
Comment (ochaloup) [reply README]: sure, changed to `-DskipDocs`

File: agent/pom.xml (hunk 2 above)
Comment (ochaloup) [reply agent/pom.xml:2380]: I did change it for empty verification file. The only thing is that the `tests/pom.xml` uses two files and I defined two separate maven properties. I can adjust if needed differently.

Comment (adinn) [reply agent/pom.xml:2380]: The new version looks fine.

**Review by adinn — APPROVED** [review]
Ok, this looks gerat. Thanks for fixing this Ondra!

**Pull request merged**